src/main/java/com/team766/controllers/CanSparkMaxSmartMotionRotationalPIDController.java

```diff
@@ -0,0 +1,34 @@
+import java.*;
+
+public class CanSparkMaxSmartMotionRotationalPIDController extends ExtendableCanSparkMaxSmartMotionPIDController {
+	
+	private final double degreesToEncoderUnitsRatio;
+	private final double antiGravK;
+
+	public CanSparkMaxSmartMotionRotationalPIDController(final String configName, final double absEncoderOffset, final double absEncoderOffsetForZeroEncoderUnits, final OffsetPoint first, final OffsetPoint second) {
+		super(configName, absEncoderOffset, absEncoderOffsetForZeroEncoderUnits, first, second);
+	}
+
+	public void updateDegreesToEncoderUnitsRatio(double ratio){
```

Comment:
it seems like this should just be set in the constructor. the conversion between units shouldn't need to change